mocks/configuration/DatabaseConfiguration.go

```diff
@@ -68,6 +71,34 @@ func (_m *DatabaseConfiguration) GetPort() int {
 	return r0
 }
 
+// GetRetry provides a mock function with given fields:
+func (_m *DatabaseConfiguration) GetRetry() uint {
```

Comment:
It's generated by [mockery](https://github.com/vektra/mockery), I can change the naming manually but it would have to be changed every time mockery is ran

Reply:
oh ok let it go so :)